kevlar/counting.py

```diff
@@ -62,77 +67,55 @@ def load_sample_seqfile(seqfiles, ksize, memory, maxfpr=0.2,
     if fpr > maxfpr:
         message += ' (FPR too high, bailing out!!!)'
         raise SystemExit(message)
-    else:
-        if outfile:
-            if not outfile.endswith(('.ct', '.counttable')):
-                outfile += '.counttable'
-            sketch.save(outfile)
-            message += '; saved to "{:s}"'.format(outfile)
-        print('[kevlar::counting]    ', message, file=logfile)
+
+    if outfile:
+        if not outfile.endswith(('.ct', '.counttable')):
```

Comment:
This behavior (with respect to filename extensions for saved sketches) should be documented somewhere. A single function to handle reading/writing sketches would help. One already exists, but it isn't suited for all cases IIRC.